### ISSUE TITLE
Button missing a text alternative

### ISSUE THREAD

**Issue by pooriamehregan** [Informasjonsforvaltning/fdk-issue-tracker#677]
[Button missing a text alternative - Issues - Accessibility NextGen - Accessibility.pdf](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/files/10402267/Button.missing.a.text.alternative.-.Issues.-.Accessibility.NextGen.-.Accessibility.pdf)
